src/Symfony/Component/Routing/Exception/MissingMandatoryParametersException.php

```diff
@@ -19,4 +19,36 @@
  */
 class MissingMandatoryParametersException extends \InvalidArgumentException implements ExceptionInterface
 {
+    private array $missingParameters = [];
+    private string $routeName = '';
+
+    public function __construct(/*array*/ $missingParameters = [], /*string*/ $routeName = '', /*int*/ $code = 0, \Throwable $previous = null)
```

Comment:
you need to add phpdoc for tools to know about the argument types

Reply:
Thanks, it's done ! (I only documented the "new" types, not the legacy ones, is that ok ?)